api/src/resolvers/banking/banking-cypher.ts

```diff
@@ -54,3 +54,26 @@ DELETE r
 
 RETURN record, church.name AS churchName, date.date AS date
 `
+export const lastButOneServiceRecord = `
```

Comment:
How many DB hits on this??

Reply:
When there's an index on serviceRecord id, it's less that 200 hits. Without the index it's in the thousands

Reply:
With the Index - 175